ova/setOVADefault.sh

```diff
@@ -38,3 +38,4 @@ echo "SHA1(${file}.ovf) = ${sha_ovf}" >> ${workspace}/new-ova/${file}.mf
 
 tar -cvf "${dest_ova}" -C "${workspace}/new-ova/" ${file}.ovf ${file}-disk-1.vmdk ${file}.mf
 rm -rf ${workspace}/new-ova/
+
```

Comment:
Remove the last line

Reply:
Done in e1ed9fda43e7bc66efb1c01f56d1564c867acdee